src/Kiota.Builder/Writers/Go/GoConventionService.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Linq;
+using Kiota.Builder.Extensions;
+
+namespace Kiota.Builder.Writers.Go {
+    public class GoConventionService : ILanguageConventionService
+    {
+        public string StreamTypeName => "[]byte";
+
+        public string VoidTypeName => string.Empty;
+
+        public string DocCommentPrefix => string.Empty;
+
+        public string PathSegmentPropertyName => "pathSegment";
+
+        public string CurrentPathPropertyName => "currentPath";
+
+        public string HttpCorePropertyName => "httpCore";
+
+        public string ParseNodeInterfaceName => "ParseNode";
+
+        public object AbstractionsHash => "ida96af0f171bb75f894a4013a6b3146a4397c58f11adb81a2b7cbea9314783a9";
+        internal string DocCommentStart = "/*";
```

Comment:
Any reasons these aren't higher up in the `ILanguageConventionService`?

Reply:
because it varies by langauge, removing as it is not in use for Go.